_layouts/docs.html

```diff
@@ -36,7 +36,7 @@
 <meta name="twitter:card" content="summary_large_image">
 <meta name="twitter:site" content="@docker_docs">
 <meta name="twitter:url" content="https://twitter.com/docker_docs">
-<meta name="twitter:title" content="Docs team">
+<meta name="twitter:title" content="Docker docs">
```

Comment:
I'm not sure how/if we can invoke that variable from the `layout`, so I'm just going with the static-fix improvement for now. (As I said in the PR description. ;) )